wordlists/names/top_male_name_germany.txt

```diff
@@ -0,0 +1,154 @@
+Aaron
+Abraham
+Adam
+Adalbert
+Adolf
+Aegidius
+Albert
+Alexander
+Alfred
+Alois
+Amand
+Andreas
+Anselm
+Anton
+August
+Augustin 
+Augustinus
+Balthasar
+Baptist
+Baruch
+Benedict
+Benjamin
+Bernhard
+Berthold/Barthold
+Bonaventur
+Bonifazius
+Carl
+Caspar
+Christian
+Christoph
+Christophorus
+Clemens
+Cleophas
+Cölestin
+Conrad
+Constantin
+Cyriakus
+Damasus
+Damian
+Daniel
+David
+Dionysius
+Dominik
+Dominikus
+Eberhard
+Eckhard
+Eduard
+Egyd
+Emanuel
+Emil
+Engelbert
+Engelhard
+Erhard
+Ernst
+Eugen
+Eustach
+Eustacius
+Ferdinand
+Fidel
+Florentius
+Franz
+Friedrich
+Fridolin
+Fritz
+Gambrich/ Gombrich
```

Comment:
Not sure what that is? Are they supposed to be two separate names or synonyms?

Reply:
these are different spellings for the same name. but to avoid confusion I am removing one spelling